include/libtorrent/kademlia/put_data.hpp

```diff
@@ -0,0 +1,94 @@
+/*
+
+Copyright (c) 2006-2015, Arvid Norberg, Thomas Yuan
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in
+      the documentation and/or other materials provided with the distribution.
+    * Neither the name of the author nor the names of its
+      contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+
+*/
+
+#ifndef PUT_DATA_HPP
```

Comment:
Sure, but I don't know which one you want.
in dht_storage.hpp
#ifndef DHT_STORAGE_HPP       (file name)

in dht_tracker.hpp
#ifndef TORRENT_DHT_TRACKER    (torrent + file name)

in dos_blocker.hpp
#ifndef TORRENT_DHT_DOS_BLOCKER    (torrent + dht + file name)

in find_data.hpp
#ifndef FIND_DATA_050323_HPP     (file name + date)

in get_item.hpp
#ifndef LIBTORRENT_GET_ITEM_HPP    (libtorrent + file name)

in node_entry.hpp
#ifndef KADEMLIA_NODE_ENTRY_HPP    (kademlia + file name)